src/lexers/vue-lexer.js

```diff
@@ -0,0 +1,25 @@
+import BaseLexer from './base-lexer'
+import JavascriptLexer from './javascript-lexer.js'
+
+export default class VueLexer extends BaseLexer {
+  constructor(options = {}) {
+    super(options)
+
+    this.functions = options.functions || ['$t']
+  }
+
+  extract(content, filename) {
+    let keys = []
+
+    const Lexer = new JavascriptLexer()
+    Lexer.on('warning', warning => this.emit('warning', warning))
+    keys = keys.concat(Lexer.extract(content))
+
+    const compiledTemplate = require('vue-template-compiler').compile(content).render
+    const Lexer2 = new JavascriptLexer({functions:this.functions})
```

Comment:
Please follow the project coding style (spaces around `{}` and after `:`)

Reply:
my ide not find lint setting in the project. can you provide settings? or "(spaces around {} and after :) " the only requirements?

Reply:
I realize that indeed there is no linter or defined coding style. We will likely introduce one soon.

In the meantine, I invite to look at how the code is currently written and follow that.